astropy/time/tests/test_fast_parser.py

```diff
@@ -142,3 +142,12 @@ class TimeYearDayTimeSubClass(TimeYearDayTime):
                 Time("2000:0601", format="yday_subclass")
     finally:
         del TimeYearDayTimeSubClass._registry["yday_subclass"]
+
+
+def test_fast_large_arrays():
+    """Test that we do not segfault on large arrays with wrong formats.
+
+    See gh-18254, where this turned out to happen with numpy 2.3.0.
+    """
+    t = Time(["J2000.0"] * 501)
```

Comment:
I think this value should be something that cannot possibly be a valid time, e.g. `'xyz'`. If there was a fast julian date parser that was the first in the guess list, then this test would stop being effective.

Reply:
OK, done.